x/sqlbuilder/predicate_clause.go

```diff
@@ -0,0 +1,164 @@
+package sqlbuilder
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"reflect"
+)
+
+type plainSQLPredicate string
+
+func (psp plainSQLPredicate) ToSQL() string { return string(psp) }
+
+func PlainSQLPredicate(exp string) PredicateClause {
+	return &staticStmtPredicateClauseWrapper{sspc: plainSQLPredicate(exp)}
+}
+
+func EqMarkers(l, r Marker) PredicateClause {
+	return &staticStmtPredicateClauseWrapper{
```

Comment:
you can do `return PlainSQLPredicate(fmt.Sprintf("%s = %s", l.ToSQL(), r.ToSQL()))`